packages/manager/apps/web/client/app/domain/dashboard/translations/Messages_en_GB.json

```diff
@@ -214,7 +214,7 @@
   "domain_tab_ZONE_no_defaut_dns_used": "You currently use the following DNS servers: ",
   "domain_tab_ZONE_set_defaut_dns": "Please use the following DNS servers, so that your DNS zone is registered:",
   "domain_tab_ZONE_title": "Here you can see the configuration of various domain name records. ",
-  "domain_tab_ZONE_info": "You can also configure these records to connect your domain to your different services (the \"Add a record\" button).",
+  "domain_tab_ZONE_info": "You can also configure these records to connect your domain to your different services (the “Add an entry” button).",
```

Comment:
**question:** Are you sure this doesn't break anything on Manager or any lint errors? Can you please check the behavior on Manager with English language?

Reply:
fixed with the requested translation